.github/ISSUE_TEMPLATE/question.md

```diff
@@ -12,4 +12,4 @@ Before asking a question, make sure you have:
 - Searched open and closed [GitHub issues](https://github.com/chaos-mesh/chaos-mesh/issues?utf8=%E2%9C%93&q=is%3Aissue)
 - Read the documentation:
   - [Chaos Mesh Readme](https://github.com/chaos-mesh/chaos-mesh)
-  - [Chaos Mesh Doc](https://github.com/chaos-mesh/chaos-mesh/tree/master/doc)
+  - [Chaos Mesh Doc](https://github.com/chaos-mesh/chaos-mesh/tree/master/website/docs)
```

Comment:
maybe use `https://chaos-mesh.org/docs` is better

Reply:
Nice idea! fixed